searchlib/src/vespa/searchlib/attribute/imported_attribute_vector_read_guard.h

```diff
@@ -26,6 +27,7 @@ class ImportedAttributeVectorReadGuard : public IAttributeVector,
 {
 private:
     using ReferencedLids = vespalib::ConstArrayRef<uint32_t>;
+    IDocumentMetaStoreContext::IReadGuard::UP _target_document_meta_store_read_guard;
```

Comment:
Consider extending class comment of ImportedAttributeVectorReadGuard describing why we take the various guards.